# Contour details
# width of the bounding box
int32  width
# height of the bounding box
int32 height
#x_min of the bounding box
int32 x_min
# x_max of the bounding box
int32 x_max
# y_min of the bounding box
int32 y_min
# y_max of the bounding box
int32 y_max
# distance from the guest
float64 distance
